docs/overview.rst

```diff
@@ -0,0 +1,39 @@
+.. _overview_label:
+
+~~~~~~~~
+Overview
+~~~~~~~~
+
+`oemoflex` provides a framework to perform a set of energy system scenarios with `oemof-solph <https://oemof-solph.readthedocs.io/>`_.
```

Comment:
Could tell in one sentence about the context in the project.

Reply:
See https://github.com/modex-flexmex/oemo-flex/pull/26#issuecomment-744547385